src/App.js

```diff
@@ -2,28 +2,37 @@ import React from 'react';
 
 import './App.scss';
 
-import posts from './api/posts';
-import comments from './api/comments';
-import users from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+import { PostList } from './components/PostList';
+
+const postsAll = postsFromServer.map(post => ({
```

Comment:
не добавляй слово `all` к переменным
```suggestion
const posts = postsFromServer.map(post => ({
```